src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
@briankendall, excuse me, don't understand yet why do you need the third value. Why just absent parameter in the command line does not work? Sure, you need the tri-state value further on, bu why it is needed in the command line?